client/scss/utilities/_root-scope-classes.scss

```diff
@@ -40,6 +40,14 @@
   }
 }
 
+@each $color-key, $color-value in $colors {
+  .font-hover--#{$color-key}:hover {
+    // stylelint-disable sh-waqar/declaration-use-variable
+    color: nth($color-value, 1);
+    // stylelint-enable sh-waqar/declaration-use-variable
+  }
+}
+
```

Comment:
Do we have anything watching the size of our CSS compiled as we have a few of these now, which are less code, but more file?